GenderPayGap.WebUI/Controllers/Admin/AdminFeedbackController.cs

```diff
@@ -57,7 +58,8 @@ public FileContentResult DownloadFeedback(bool nonSpamOnly = false)
             var memoryStream = new MemoryStream();
             using (var writer = new StreamWriter(memoryStream))
             {
-                using (var csv = new CsvWriter(writer, CultureInfo.CurrentCulture))
+                var config = new CsvConfiguration(CultureInfo.CurrentCulture) { ShouldQuote = (s, context) => true, TrimOptions = TrimOptions.InsideQuotes, SanitizeForInjection = true};
+                using (var csv = new CsvWriter(writer, config))

```

Comment:
Pretty much the only important bit - which I overlooked in the first commit because I thought everywhere would use the same sensible config! Turns out the `ToCsv` function which sets the config isn't used for the feedback download, which maybe seems like something to refactor in future (or here, if you think it's worth it). 
Having tested, this now works - excel injection just shows the text of the attempted query instead of doing anything dangerous with it!

Reply:
Ahhh great spot!

I'd be in favour of refactoring _all_ the CSV readers to use a standard CsvConfiguration which has SanitizeForInjection set to true. That way future CSV exports don't need to remember to add it in and it just happens automatically. Is that a thing we can do?

(Do we use the other ones eg ShouldQuote, TrimOptions everywhere too? could just inject a consistent configuration to everything)

Reply:
Yeah, agreed that that's probably the best way to go with this. All the CSV readers _don't_ use the same config though - I've just realised that I should probably remove ShouldQuote and TrimOptions from here since they weren't set here originally!